src/KawaBot.Tests/BasicControllerTests.fs

```diff
@@ -4,20 +4,36 @@ open System.Net
 open KawaBot.Feeder
 open KawaBot.Server
 open NUnit.Framework
+open Moq
+open System.Net.Http
+open System.Text
+open Newtonsoft.Json
 
 [<TestFixture>]
 [<Category(Categories.Integration)>]
 type BasicControllerTests =
     val private _server: Http.HttpListener
     val private _feeder: Feeder
+    val private _httpRequsterMock: Mock<IHTTPRequester>
 
     new() as this =
         {
             _server = new Http.HttpListener("http://*:80/")
             _feeder = new Feeder(80)
+            _httpRequsterMock = Mock<IHTTPRequester>()
         } then
             HealthController.GetUrlHandler()
             |> this._server.Register
+            SlackVerificationController.GetUrlHandler()
+            |> this._server.Register
+            let oauth = new OAuthController(this._httpRequsterMock.Object)
+            oauth.GetUrlHandler() |> this._server.Register
+
+    member this.StubOAuthController(body: string) =
+        let resp = new HttpResponseMessage()
+        resp.Content <- new ByteArrayContent(Encoding.UTF8.GetBytes(body))
+        this._httpRequsterMock.Setup(fun req -> req.HTTPGet(It.IsAny<string>()))
```

Comment:
Lets use the Pascal case for abbreviations longer than 3 chars: `HTTP` -> `Http`.